src/components/CustomTextInput.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { ReactElement, useState } from 'react'
+import { TextInputProps } from 'react-native'
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled, { useTheme } from 'styled-components/native'
+
+import { CloseIcon } from '../../assets/images'
+import theme from '../constants/theme'
+import { ContentError } from './text/Content'
+
+const LINE_HEIGHT = 30
+const MIN_HEIGHT = 60
+
+const StyledTextInput = styled.TextInput`
+  font-size: ${props => props.theme.fonts.defaultFontSize};
+  font-weight: ${props => props.theme.fonts.lightFontWeight};
+  letter-spacing: ${props => props.theme.fonts.listTitleLetterSpacing};
+  font-family: ${props => props.theme.fonts.contentFontRegular};
+  color: ${prop => prop.theme.colors.primary};
+  flex: 1;
+`
+
+const ClearContainer = styled.TouchableOpacity`
+  width: ${wp('6%')}px;
+  height: ${wp('6%')}px;
+`
+
+const TextInputContainer = styled.View<{ lines: number; borderColor: string; showErrorValidation: boolean }>`
+  border: 1px solid ${props => props.borderColor};
+  padding-left: ${props => props.theme.spacings.sm};
+  padding-right: ${props => props.theme.spacings.sm};
```

Comment:
(optional) can be simplified to 
`padding: 0 ${props => props.theme.spacings.sm}`